exporters/jaeger/src/main/java/io/opentelemetry/exporters/otproto/TraceProtoUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.exporters.otproto;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.protobuf.ByteString;
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.TraceId;
+
+/** Utilities for converting various objects to protobuf representations. */
+public class TraceProtoUtils {
```

Comment:
Please just create an issue (my suggestion was different):
exporters/jaeger/src/main/java/io/opentelemetry/exporters/otproto/TraceProtoUtils.java -> exporters/otproto/src/main/java/io/opentelemetry/exporters/otproto/TraceProtoUtils.java

We can do that later.

Reply:
#591